staff/emma-cowdery/gamma-template/about.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+        crossorigin="anonymous">
+</head>
+
+<body class="body-about">
+    <header class="nav-bar">
+        <p class="nav-bar__gamma">Gamma</p>
+        <a href="./home.html" class="nav-bar__components">Home</a>
+        <p class="nav-bar__components nav-bar__components__about">About</p>
+        <a href="./contact.html" class="nav-bar__components">Contact</a>
+        <i class="fas fa-bars nav-bar__hamburguer"></i>
+    </header>
+    <main>
+        <section class="page-title">
+            <h1 class="page-title__header">About Gamma</h1>
+        </section>
+        <section class="main-grid-container">
+            <article class="main-grid-container__about">
+                <h2 class="main-grid-container__about__title">intrigueproof</h2>
```

Comment:
Each class should have a maximum of 1 child. You can use - for grandchildren! Ex: Class__child-grandchild